### PR TITLE
Returning generated StaticBody of MeshInstance3D

### PR DESCRIPTION
**Pull request by CoreTaxxe**
PR request for https://github.com/godotengine/godot-proposals/issues/5567 returning generated StaticBody colliders for 
`create_convex_collision()` `create_multiple_convex_collisions()` and `create_trimesh_collision()`

### CODE REVIEW

File: scene/3d/mesh_instance_3d.h [hunk 1]
```diff
@@ -97,6 +99,9 @@ class MeshInstance3D : public GeometryInstance3D {
 
 	MeshInstance3D();
 	~MeshInstance3D();
+
+private:
+    String default_collider_suffix = "_staticbody3D";
```
Comment (RedMser) [inline scene/3d/mesh_instance_3d.h:104]: Not sure, but I can imagine the old `_col` suffix was chosen intentionally to [fit the naming convention](https://docs.godotengine.org/en/latest/tutorials/assets_pipeline/importing_scenes.html#create-collisions-col-convcol-colonly-convcolonly) of importing collision meshes?

**Comment by CoreTaxxe** [comment]
accidentally pushed

**Comment by akien-mga** [comment]
For the record, the PR was closed automatically by GitHub because you seem to have pushed the latest `master` branch on your PR branch, without your original work on top of it. So there was nothing left to merge and in this case GitHub force closes PRs.

You may want to redo a new PR after reapplying your changes (if you lost them in the process, you can check `git reflog` to reset to the original commit again).

Apologies for the very long delay providing feedback on this PR, we're very short on contributors in the physics area.

**Comment by CoreTaxxe** [comment]
> For the record, the PR was closed automatically by GitHub because you seem to have pushed the latest `master` branch on your PR branch, without your original work on top of it. So there was nothing left to merge and in this case GitHub force closes PRs.
> 
> You may want to redo a new PR after reapplying your changes (if you lost them in the process, you can check `git reflog` to reset to the original commit again).
> 
> Apologies for the very long delay providing feedback on this PR, we're very short on contributors/reviewers in the physics area.

Yeah that was intentional! Too much changed and most of the changes were/are already implemented. (except for multiple collisions)

Thanks for the reply tho, appreciate it

**Pull request closed without merging**